StickKnightsOnline-Client/main.cpp

```diff
@@ -8899,3 +8989,5 @@ void Kill()
 	// at the time of writing SDL_Quit() causes a crash upon exit.
 	exit(0);
 }
+
```

Comment:
Remove whitespace, but if you added a newline to the end of the file that's alright.